cmd/genkey.go

```diff
@@ -0,0 +1,128 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/gofrs/uuid"
+	"github.com/kairos-io/enki/pkg/config"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"k8s.io/mount-utils"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+func NewGenkeyCmd() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "genkey NAME",
+		Short: "Generate secureboot keys under the uuid generated by NAME",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cobraCmd *cobra.Command, args []string) error {
+			// Set this after parsing of the flags, so it fails on parsing and prints usage properly
+			cobraCmd.SilenceUsage = true
+			cobraCmd.SilenceErrors = true // Do not propagate errors down the line, we control them
+
+			cfg, err := config.ReadConfigBuild(viper.GetString("config-dir"), cobraCmd.Flags(), &mount.FakeMounter{})
+			if err != nil {
+				return err
+			}
+			l := cfg.Logger
+			uid := uuid.NewV5(uuid.NamespaceDNS, args[0])
+			output, _ := cobraCmd.Flags().GetString("output")
+
+			err = os.MkdirAll(output, 0755)
+			if err != nil {
+				l.Errorf("Error creating output directory: %s", err)
+				return err
+			}
+
+			for _, t := range []string{"PK", "KEK", "DB"} {
```

Comment:
silly ask but since t extends so long I wouldn't mind calling it something a bit more meaningful

Reply:
maybe now its better? There is not a really good name to use for this that doesnt lead to confusion....